### PR TITLE
Jugner Forest LUA cleanup/style

### PR DESCRIPTION
**Pull request by wrenffxi**
* signposts identified by ID/offset rather than X/Y
* Meteormauler Zhagtegg added 2HR and damage reduction
* npcUtilified trades
* style guide

### CODE REVIEW

**Comment by TeoTwawki** [comment]
we'd rather you didn't put multiple things in one pr, much less in the same commit (plus style changes to boot), when doing these things. its makes it harder to review.

Also using coordinates in and of itself for the signposts isn't necessarily a bad thing - future proofs them from ID changes.

File: scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua [hunk 1]
```diff
@@ -2,6 +2,12 @@
 -- Area: Jugner Forest
 --   NM: Meteormauler Zhagtegg
 -----------------------------------
+mixins = {require("scripts/mixins/job_special")}
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMod(dsp.mod.DMG, -50) -- "Has an innate damage reduction trait, about 50%"
```
Comment (TeoTwawki) [inline scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua]: clopededia is wrong here. he had magic dmg redux only (as bg correctly shows)
https://www.bg-wiki.com/bg/Meteormauler_Zhagtegg

Comment (TeoTwawki) [reply scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua]: Incidentally, the entire line of similar NMs all have either phys or mag dmg reduction and a posse (auto follows them) that does not always pop (separate timers maybe, who knows)

the quadav gets -50 physical

Comment (TeoTwawki) [reply scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua]: (which I can't prove atm wiki's don't list, just that one anecdotal comment on every NM..from a blu)

Comment (TeoTwawki) [reply scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua]: possibly their resistance is related to their posse being up, no way to know atm

**Comment by wrenffxi** [comment]
will split into multiple PRs

**Pull request closed without merging**